feg/gateway/services/n7_n40_proxy/servicers/session_controller.go

```diff
@@ -70,8 +63,20 @@ func (srv *CentralSessionController) CreateSession(
 	ctx context.Context,
 	request *protos.CreateSessionRequest,
 ) (*protos.CreateSessionResponse, error) {
+	if err := validateCreateSessionRequest(request); err != nil {
+		return nil, err
+	}
 
-	return (&protos.UnimplementedCentralSessionControllerServer{}).CreateSession(ctx, request)
+	smPolicyDecision, smPolicyId, err := srv.getSmPolicyRulesIfEnabled(request)
```

Comment:
`smPolicyDecision` is a weird name that makes me think there is more than a policy there. But checking the definition [here](smPolicyDecision) I can see it is just a policy. 

What about if we use easier names like 
```
policy, policyId, err := srv.getSmPolicyRulesIfEnabled(request)
```

Adding the SM everywhere may be overkill

Reply:
The logic behind adding sm prefix is to make it clear the type is a N7 type and to differentiate it from a proto type. However will change the name as suggested.